pulp_ansible/tests/functional/api/collection/v3/test_sync.py

```diff
@@ -179,8 +180,8 @@ def test_sync_with_token_from_automation_hub(self):
 
 
 @unittest.skipUnless(
-    "CI_AUTOMATION_HUB_TOKEN_AUTH" in os.environ,
-    "'CI_AUTOMATION_HUB_TOKEN_AUTH' env var is not defined",
+    "QA_AUTOMATION_HUB_TOKEN_AUTH" in os.environ,
+    "'QA_AUTOMATION_HUB_TOKEN_AUTH' env var is not defined",
```

Comment:
These are being renamed in github secrets right?

Reply:
this is not being tested in github actions because it is behind a vpn